### DISCUSSION TITLE
Keeping Prometheus server running

### DISCUSSION THREAD

**Discussion by mwatilenhu** [prometheus/docs#2271] [Q&A]
Hi guys,

So I downloaded Prometheus for windows, we are using it with postilion. I just want to know how to keep it up and running.
Because at the moment I have to log onto the server, and run the .exe file every time I need to access the interface via the browser.
Is there no way to keep it up and running and we don't necessarily have to log onto the server itself but rather just use a url to access the dashboard(s)? The .exe file opens a cmd window which must stay open, once I close the window or log off from the server the window closes.

I don't see anything in the documentation for this.

Thank You

**Comment by RichiH** [comment]
I don't know anything about Windows, sorry.

But I wanted to ask: Are saying https://www.der-postillon.com/ runs with it?

**Reply by mwatilenhu** [reply]
Hi, 

So Postilion is a banking system for Cards and Card Transactions management: https://www.aciworldwide.com/solutions/postilion
It has the capabilities to integrate with Prometheus for monitoring and such. I have configured the JMX exporters for the different modules that must be monitored, the challenge I am having is keeping Prometheus up and running on the server.

**Reply by mwatilenhu** [reply]
Hi, 

I got a solution, I will use the docker image. 👍🏾 

Thanks
